tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Run Data App in Postman
+summary: Learn how to run your Data App in Postman.
+---
+
+# Run Data App in Postman
+
+[Postman](https://www.postman.com/) is an API platform for building and using APIs, which simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
+
+In TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service), you can easily import your Data App to Postman and leverage Postman's extensive tools to enhance your API development experience.
+
+This document describes how to import your Data App to Postman and how to run your Data App in Postman.
+
+## Before you begin
+
+Before importing a Data App to Postman, make sure that you have the following:
+
+- A [Postman](https://www.postman.com/) account.
+- A [Postman desktop app](https://www.postman.com/downloads) (Optional. You can use the Postman webpage version without downloading the app.)
+- A [Data App](/tidb-cloud/data-service-manage-data-app.md) with at least one [endpoint](/tidb-cloud/data-service-manage-endpoint.md). Only endpoints that meet the following requirements can be imported to Postman:
+
+    - The target cluster is selected.
+    - The endpoint path and request method are configured.
+    - The SQL statements are written.
```

Comment:
I am a bit concerned calling it out especially "The SQL statement are written", reason is that tomorrow there may be custom functions just like SQL then this statement alone may not hold good.

Reply:
@jusong PTAL